tensorflow/core/common_runtime/gpu/gpu_init.h

```diff
@@ -24,8 +24,17 @@ class Platform;
 
 namespace tensorflow {
 
-// Initializes the CUDA platform and returns OK if the CUDA
-// platform could be initialized.
```

Comment:
Add a comment like this here:

// The choice of which kind of gpu that the TF GPU device corresponds to is a compile-time decision,
// and thus we cannot build a single binary that supports multiple kinds.  This is sub-optimal, and a
// better solution will likely need to be developed in the future.  Also see the commentary here:
// https://github.com/tensorflow/tensorflow/pull/20845

Reply:
@tatatodd thanks. I've just pushed a new commit to add the comment